shell/platform/android/io/flutter/view/AccessibilityViewEmbedder.java

```diff
@@ -220,6 +225,122 @@ private void copyAccessibilityFields(AccessibilityNodeInfo input, AccessibilityN
         }
     }
 
+    /**
+     * Delegates an AccessibilityNodeProvider#requestSendAccessibilityEvent from the AccessibilityBridge to the embedded
+     * view.
+     *
+     * @return True if the event was sent.
+     */
+    public boolean requestSendAccessibilityEvent(View embeddedView, View eventOrigin, AccessibilityEvent event) {
```

Comment:
subjective, optional: It's verbose but in other places we've been adding in `@NonNull` annotations in addition to `@Nullable`. So please tag the potentially null params and members in this patch either way depending on if they're allowed to be null. This applies to a couple places throughout.

Reply:
Thanks for keeping up with the nullability tags.  They're easy to forget, but I think we definitely want them on every non-primitive input and output.

Reply:
done